lib/osf-components/addon/components/search-page/styles.scss

```diff
@@ -164,43 +217,87 @@
 
 
 .object-type-filter-link {
-    padding: 10px 15px;
     font-size: 1.3em;
+    padding: 18px 15px;
 
     &:global(.active),
     &:hover {
-        text-decoration: none;
-        background-color: $color-bg-gray;
+        background-color: $color-bg-white;
+        border-bottom: 1px solid $color-text-black;
         color: $color-text-black;
-        border-bottom: 2px solid $color-text-black;
+        text-decoration: none;
     }
 }
 
 .sort-dropdown {
-    width: 170px;
-    margin-top: 10px;
+    background-color: $color-bg-gray-blue-light;
+
+    p {
+        margin: 0 0 0 2px;
+    }
+
+    div {
+        background-color: $color-bg-gray-blue-light;
+        border: 0 solid transparent;
+        color: $color-bg-blue-dark;
+        width: 170px;
+    }
+
+    > div {
+        display: flex;
+    }
+}
+
+.ember-power-select-status-icon {
+    right: 10px;
+    top: 12px;
```

Comment:
Without doing `:global(.ember-power-select-status-icon)`, this won't change the style for the dropdown's caret. If this is still needed, I would nest it into `.sort-dropdown :global(...)` so the style doesn't leak into other places where we use power-select

Reply:
Styling for the dropdown's caret is in effect with the current code. The global pseudoclass CSS module should not be needed:

<img width="1626" alt="image" src="https://github.com/CenterForOpenScience/ember-osf-web/assets/82047646/6b2bc66c-4e46-41b4-b687-004bcc4559c7">

<img width="1630" alt="image" src="https://github.com/CenterForOpenScience/ember-osf-web/assets/82047646/5d3fdb1a-4942-47dd-9c83-fd6875e04a9b">


however I have moved it to the sort-dropdown local class selector's block to prevent any global styling. The styling remains the same but I agree with the scope being within the dropdown.